samples/snippets/noxfile.py

```diff
@@ -38,25 +38,28 @@
 
 TEST_CONFIG = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7"],
+    'ignored_versions': ["2.7"],
```

Comment:
@crwilcox I'm a bit curious as to why the "blacken" gets reverted. We run `blacken` as the last step here: https://github.com/googleapis/python-bigquery-reservation/blob/master/synth.py#L66

But maybe we need to loop through the samples directories and run those blacken sessions?

Reply:
Reverting for now. Can discuss in https://github.com/googleapis/python-bigquery-reservation/pull/66 which contains only this change.